### PR TITLE
feat!: update icons to all use icon-module

### PR DESCRIPTION
**Pull request by luqven**
## Before this PR
Icon styles varied unpredictably and could not be overridden easily where used.

## After this PR
All icons accept an optional `className` prop and use the `styles.container` stylesheet from  Icon.module.scss` to standardize the styling.

### CODE REVIEW

**Comment by luqven** [comment]
Current dependencies on/for this PR:
* next
  * **PR #235** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/235" target="_blank">Graphite</a> 
    * **PR #236** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/236" target="_blank">Graphite</a> 
      * **PR #237** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/237" target="_blank">Graphite</a>  (this PR)

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

File: frontend/src/components/icons/AddSvg.tsx [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
-  return (
-    <div className={styles.container}>
-      <svg viewBox="0 0 16 16">
-        <polygon points="15,6 10,6 10,1 6,1 6,6 1,6 1,10 6,10 6,15 10,15 10,10 15,10 " />
-      </svg>
-    </div>
-  );
-};
+export const AddSvg = ({ className }: { className?: string }) => (
+  <div className={`${styles.container} ${className}`}>
```
Comment (frederickfogerty) [inline frontend/src/components/icons/AddSvg.tsx:5]: I think in this case if className is `undefined` then the rendered className will be `container undefined`. Therefore here I think you always need to do `${className || ''}`

**Review by frederickfogerty — APPROVED** [review]
LGTM

File: frontend/src/components/icons/AddSvg.tsx (hunk 1 above)
Comment (luqven) [reply frontend/src/components/icons/AddSvg.tsx:5]: 💯 will change and fix.

**Pull request merged**